packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -14,7 +14,7 @@ const { pregQuote } = require('../../string-utils-common');
 import { MarkupToHtml } from '@joplin/renderer';
 
 export default class InteropService_Importer_Md extends InteropService_Importer_Base {
-	private importedNotes: Record<string, NoteEntity> = {};
+	importedNotes: Record<string, NoteEntity> = {};
```

Comment:
Is it still needed? As I couldn't find any reference to this property.

Reply:
This is used later in the module, it keeps track of imports so that we don't end up with an import cycle.